salt/modules/x509.py

```diff
@@ -1351,6 +1351,18 @@ def create_certificate(
 
         The above signing policy can be invoked with ``signing_policy=www``
 
+    not_before:
+        Initial validity date for the certificate. This date must be specified
+        in the format '%Y-%m-%d %H:%M:%S'.
+
+        .. versionadded:: Neon
```

Comment:
these versionadded will now need to be `Sodium` :)

Reply:
Hey @Ch3LL, fair enough, version changed!